### PR TITLE
Add display test 🎅🏽

### LINKED ISSUE #143: manifest `display` property
Do we want to suggest or require any specific display property for PWA?

The options, thanks to a [great presentation](https://dev.opera.com/blog/pwa-taipei/) by @brucelawson

### `"display": "standalone"`
image

### `"display": "fullscreen"`
image

### `"display": "browser"` (the [default](https://www.w3.org/TR/appmanifest/#dfn-fallback-display-mode))
image

`display: minimal-ui` also exists in the spec, though I'm not sure Chrome supports it.

<hr>

IMO, `standalone` is the preferred UX. (Though I can see uses of `fullscreen`)

Should we require either `standalone` or `fullscreen`, with a strong preference for `standalone`?

wdyt, @owencm @tjsavage?

### PR DESCRIPTION
**Pull request by samccone**
Fixes #143 🐐 💃

### CODE REVIEW

**Comment by brendankenny** [comment]
LGTM

File: audits/mobile-friendly/display.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+  /**
+   * @override
+   */
+  static get description() {
+    return 'Manifest has suggested display property';
+  }
+
+  /**
+   * @param {string | null} val
```
Comment (brendankenny) [inline audits/mobile-friendly/display.js]: extra spaces

**Comment by brendankenny** [comment]
🔍 💻 ✔️

**Comment by paullewis** [comment]
Just started reviewing :)

**Pull request closed without merging**